server/utils/chooseScraper.js

```diff
@@ -0,0 +1,11 @@
+const chooseScraper = (url) => {
+	if (url.includes("amazon")) {
```

Comment:
a switch statement instead of a bunch of if-else seems appropriate here :)

Reply:
is there a way to use switch in here since I am using ```url.includes()``` as the condition?